docs/data/sources/google-tag-manager-client-legacy.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Amplitude Analytics Legacy
+description: Official Google Tag Manager template for Amplitude Analytics for data collection.
+---
+
+Client-side Google Tag Manager Template for Amplitude Analytics. The tag utilizes the [Amplitude JavaScript SDK](../../data/sdks/javascript/index/) for data collection.
```

Comment:
```suggestion
This is the client-side Google Tag Manager Template for Amplitude Analytics. The tag utilizes the [Amplitude JavaScript SDK](../../data/sdks/javascript/index/) for data collection.
```

Reply:
Added "This is the" to make it a complete sentence and applied the same changes for all GTM template pages.